Core/Src/subsystems/gimbal.cpp

```diff
@@ -57,19 +57,22 @@ void act() {
     }
 }
 
-double calculateAngleError(double currAngle, double targetAngle) {
-    // IMPLEMENT THIS
-    /* 
-    Given a current angle value (currAngle, in radians) and a target angle value (targetAngle, in radians):
-        - Find and return the shortest angle error in radians with direction.
-    (hint hint use trig functions)
-
-    EX: If our current angle is 359 degrees, and the target angle is 2 degrees, this function
-        should return 3 degrees, as the shortest path from the current angle to the target angle
-        will be 3 degrees clockwise.
-        (the example was in degrees, please use radians)
-    */
-    return 0;
+double calculateAngularPath(double currAngle, double targetAngle) {
+	//Positive is counterclockwise while Negative is clockwise
+    double angleDelta = currAngle - targetAngle;
+	
+	if (fabs(angleDelta <= PI)) //Check if within simple case of within 180 degrees
```

Comment:
Might have the parentheses wrong for fabs, right now it doesn't make sense to me